include/mata/nfa.hh

```diff
@@ -247,7 +251,20 @@ public:
     size_t size() const;
 
 
-    Post & operator[] (State q)
+    //Post & operator[] (State q)
```

Comment:
Is it worth keeping the code commented out?

Reply:
no, it was a working version, actually I don't know why the reviewers were assigned automatically

Reply:
I assigned them manually.